packages/office-ui-fabric-react/src/components/MessageBar/examples/MessageBar.Basic.Example.tsx

```diff
@@ -1,160 +1,195 @@
+      condimentum mauris.
+    </MessageBar>
+  );
+
+  const SevereExample = () => (
+    <MessageBar
+      messageBarType={MessageBarType.severeWarning}
+      actions={
+        <div>
+          <MessageBarButton>Yes</MessageBarButton>
+          <MessageBarButton>No</MessageBarButton>
+        </div>
+      }
+    >
+      SevereWarning MessageBar with action buttons which defaults to multiline.
+      <Link href="www.bing.com" target="_blank">
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
+
+  const SuccessExample = () => (
+    <MessageBar
+      actions={
+        <div>
+          <MessageBarButton>Yes</MessageBarButton>
+          <MessageBarButton>No</MessageBarButton>
+        </div>
+      }
+      messageBarType={MessageBarType.success}
+      isMultiline={false}
+    >
+      Success MessageBar with single line and action buttons.
+      <Link href="www.bing.com" target="_blank">
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
+
+  const WarningExample = () => (
+    <MessageBar
+      messageBarType={MessageBarType.warning}
+      isMultiline={false}
+      onDismiss={resetChoice}
+      dismissButtonAriaLabel="Close"
+      actions={
+        <div>
+          <MessageBarButton>Action</MessageBarButton>
+        </div>
+      }
+    >
+      Warning MessageBar content.
+      <Link href="www.bing.com" target="_blank">
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
+
+  const WarningExample2 = () => (
+    <MessageBar
+      onDismiss={resetChoice}
+      dismissButtonAriaLabel="Close"
+      messageBarType={MessageBarType.warning}
+      actions={
+        <div>
+          <MessageBarButton>Yes</MessageBarButton>
+          <MessageBarButton>No</MessageBarButton>
+        </div>
+      }
+    >
+      <b>Warning defaults to multiline</b>. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi luctus, purus a lobortis
+      tristique, odio augue pharetra metus, ac placerat nunc mi nec dui. Vestibulum aliquam et nunc semper scelerisque. Curabitur vitae orci
+      nec quam condimentum porttitor et sed lacus. Vivamus ac efficitur leo. Cras faucibus mauris libero, ac placerat erat euismod et. Donec
+      pulvinar commodo odio sit amet faucibus. In hac habitasse platea dictumst. Duis eu ante commodo, condimentum nibh pellentesque,
+      laoreet enim. Fusce massa lorem, ultrices eu mi a, fermentum suscipit magna. Integer porta purus pulvinar, hendrerit felis eget,
+      condimentum mauris.
+      <Link href="www.bing.com" target="_blank">
+        Visit our website.
+      </Link>
+    </MessageBar>
+  );
+
+  return (
+    <Stack {...horizontalStackProps}>
+      <StackItem disableShrink>
+        <ChoiceGroup
+          label="Select a MessageBar Example Below"
+          selectedKey={choice}
+          // tslint:disable-next-line: jsx-no-lambda
+          onChange={(e, v) => setChoice(v!.key)}
+          options={[
```

Comment:
These options should probably also go in a const outside the body of the component, so the array doesn't mutate every re-render.